scenes/Game/Game.tscn

```diff
@@ -0,0 +1,70 @@
+[gd_scene load_steps=3 format=2]
+
+[ext_resource path="res://scenes/Game/Game.gd" type="Script" id=1]
+[ext_resource path="res://themes/start_menu.tres" type="Theme" id=2]
```

Comment:
@j-rodriguez99 I believe this is where the font for the menu buttons are coming from. I'm using the same theme that you created.